packages/office-ui-fabric-react/src/components/Nav/Nav.styles.ts

```diff
@@ -130,15 +129,15 @@ export const getStyles = (props: INavStyleProps): INavStyles => {
       },
       !isDisabled && {
         selectors: {
-          '.ms-Nav-compositeLink:hover &': {
-            backgroundColor: palette.neutralLighterAlt,
+          '$compositeLink:hover &': {
```

Comment:
The [latest recommendation](https://github.com/OfficeDev/office-ui-fabric-react/tree/master/packages/merge-styles#referencing-child-elements-within-the-mergestylesets-scope) from David is to avoid using the `$` selector and favor the global classnames that are static and unchanged

Reply:
Oh interesting, good to know.